_1327/documents/templates/editor.html

```diff
@@ -2,231 +2,305 @@
 {% load i18n %}
```

Comment:
It's really hard to see what you changed in this file. Please separate formatting and code changes in different commits or pull requests.